lib/htmllint.js

```diff
@@ -92,9 +92,9 @@ function htmllint(config, done) {
 
     const files = config.files.map(path.normalize);
 
-    async.mapSeries(chunkify(files, maxChars), (chunk, cb) => {
+    async.mapSeries(chunkify(files, MAX_CHARS), (chunk, cb) => {
       exec(cmd(java, chunk, config.noLangDetect), {
-        maxBuffer
+        MAX_BUFFER
```

Comment:
This is wrong. The option must still be passed as `maxBuffer`: https://nodejs.org/dist/latest-v10.x/docs/api/child_process.html#child_process_child_process_execfile_file_args_options_callback

Reply:
Ah you are right, needs to be `maxBuffer: MAX_BUFFER`

Reply:
Pushed